### ISSUE TITLE
Edit Service Road preset

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/iD#7586]
There are only 5 options on the list, could we make them translatable?
![highway_service](https://user-images.githubusercontent.com/57260466/81506177-cb608f80-92f4-11ea-8035-1929b2953826.png)

And reorder them? Because for example `service=driveway` is used much more often than `service=parking_aisle`.

Current usage:
```
driveway		7 227 827
parking_aisle		4 250 522
alley			1 190 037
drive-through		104 974
emergency_access	26 189
```

**Comment by manfredbrandl** [comment]
I remember that translating iD Version 3 will be offered later, because Translators want to concentrate now on the productive iD-Version and there are too many changes in development now.
Having an order of options which reflects the usage numbers is usefull anyway.

**Comment by quincylvania** [comment]
We currently can't make combo fields translatable without limiting the options to a fixed list (see #2708). There are all kinds of service roads people might make up tags for, so it's important to allow custom values here.
